webplugin/js/app/mck-sidebox-1.0.js

```
@@ -2968,54 +2875,16 @@ const firstVisibleMsg = {
                                     'Start Conversation';
                                 submitBtn.innerHTML = submitLabel;
                                 submitBtn.setAttribute('aria-label', submitLabel);
-                                if (shouldShowLivechatAuthForm || isPreLeadEnabled) {
-                                    submitBtn.classList.remove('n-vis');
-                                    submitBtn.removeAttribute('disabled');
-                                }
+                                submitBtn.classList.remove('n-vis');
+                                submitBtn.removeAttribute('disabled');
                             }
                             var loginErrorNode = document.getElementById('km-error-chat-login');
```

Biome (2.4.6)

[error] Shouldn't redeclare 'loginErrorNode'. Consider to delete it or rename it.

(lint/suspicious/noRedeclare)